snowflake/cortex/BUILD.bazel

```diff
@@ -161,6 +199,7 @@ py_library(
     deps = [
         ":classify_text",
         ":complete",
+        ":embed_text_768",
         ":extract_answer",
```

Comment:
I think `":embed_text_1024",` is needed here as well

There is a `package_visibility_test.py` file in this folder -- is it possible to add a test there? It helps to catch things like this

Reply:
Thanks for catching this, fixed.